### ISSUE TITLE
Culture and Hostnames inputs do not have a label

### ISSUE THREAD

**Issue by lordscarlet** [umbraco/Umbraco-CMS#8724]
Under _Culture and Hostnames_ for a root node, the _Domain_ and _Language_ inputs do not have an accessible label

## Umbraco version

I am seeing this issue on Umbraco version: 8.6.1

Reproduction
------------

If you're filing a bug, please describe how to reproduce it. Include as much
relevant information as possible, such as:

### Bug summary

The _Domain_ and _Language_ inputs do not have an accessible name

### Steps to reproduce

1. Navigate to the _Content_ section
2. View the actions on a root node
3. Choose _Culture and Hostnames_

### Expected result

A screen reader should be able to view the name for each input field, typically via a `<label>` with a `for` attribute.

### Actual result

The input fields do not have an associated label

**Comment by umbrabot** [comment]
Hi @lordscarlet,

We're writing to let you know that we would love some help with this issue. We feel that this issue is ideal to flag for a community member to work on it. Once flagged here, folk looking for issues to work on will know to look at yours. Of course, please feel free work on this yourself ;-).  If there are any changes to this status, we'll be sure to let you know. 

For more information about issues and states, have a look at [this blog post](https://umbraco.com/blog/the-umbraco-issue-tracker-process/)

Thanks muchly, from your friendly Umbraco GitHub bot :-)

**Comment by umbrabot** [comment]
Closing this in relation to PR #8726 - make sure to read the close reason on #8726
